src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1656,6 +1661,8 @@ private static ParameterSymbol GetParameter(int argIndex, MemberAnalysisResult r
 
             // We might have got out of the loop above early and allSame isn't completely calculated.
             // We need to ensure that we are not going to skip over the next 'if' because of that.
+            // This condition requires the methods being compared have parameters of identical types but different ref kinds.
```

Comment:
>This condition requires [](start = 15, length = 23)

The condition doesn't require that and we don't really know if there are other scenarios when condition could be true, consider changing the comment to say: "For example, we can get in this situation when ..."

Reply:
Done.